src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx

```diff
@@ -39,7 +39,12 @@ describe('the useLocalTracks hook', () => {
       });
 
       expect(Video.createLocalTracks).toHaveBeenCalledWith({
-        audio: true,
+        audio: {
+          noiseCancellationOptions: {
+            sdkAssetsPath: '/noisecancellation',
+            vendor: 'krisp',
+          },
```

Comment:
It seems like we re-use this multiple times in this file. Can we store this as a variable once and reuse that?

Reply:
@PikaJoyce I'll do that in a separate PR. Now, I'm focused on getting this out.